exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Op.kt

```diff
@@ -683,6 +684,9 @@ fun stringParam(value: String): Expression<String> = QueryParameter(value, TextC
 /** Returns the specified [value] as a decimal query parameter. */
 fun decimalParam(value: BigDecimal): Expression<BigDecimal> = QueryParameter(value, DecimalColumnType(value.precision(), value.scale()))
 
+/** Returns the specified [value] as a blob query parameter. */
+fun blobParam(value: ExposedBlob): Expression<ExposedBlob> = QueryParameter(value, BlobColumnType())
```

Comment:
Could you provide a bit more context on why this is needed, please?

Does it fixes a bug? Does it addresses a particular issue you're having? 
Thanks!

Reply:
The context is right there in the file. Every column type (except blob which is an oversight) has a `<type>Literal()`, `<type>Param()`, `class <type>ColumnType` triple. This is simply restoring consistency.